rskj-core/src/test/java/co/rsk/peg/federation/PowpegMigrationTest.java

```diff
@@ -198,8 +187,8 @@ private void testChangePowpeg(
         federationStorageProvider.save(btcParams, activations);
 
         // Proceed with the powpeg change
-        FederationSupportImpl federationSupport = new FederationSupportImpl(bridgeConstants.getFederationConstants(), federationStorageProvider, initialBlock, activations);
-        federationSupport.commitFederation(false, pendingFederation.getHash(), bridgeEventLogger);
+        FederationSupportImpl federationSupportImpl = new FederationSupportImpl(bridgeConstants.getFederationConstants(), federationStorageProvider, initialBlock, activations);
+        federationSupportImpl.commitFederation(false, pendingFederation.getHash(), bridgeEventLogger);
```

Comment:
here we need to create a federation support implementation to can call the commitFederation method (that does not exist in the interface)

Reply:
Why this method is not included as part of the interface? Isn't the `commitFederation` an operation done by a `federationSupport` type?

Reply:
the `commitFederation` method is an [internal operation](https://github.com/rsksmart/rskj/blob/federation-support-refactor-integration/rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java#L488) done after voting the commit function to be performed by the bridge

Reply:
is the only one that is `protected` (instead of `private`) so we can test it with less complexity